test/helpers/workflow.js

```diff
@@ -31,10 +31,10 @@ module.exports = {
     return knex.destroy();
   },
 
-  resetDBs: () => {
+  resetDBs: ({ keepalive = false } = {}) => {
     return Promise.resolve()
       .then(() => tfdb.reset())
-      .then(() => aslDb(settings.db).init(fixtures.default));
+      .then(() => aslDb(settings.db).init(fixtures.default, keepalive));
```

Comment:
What is this for?

Reply:
It keeps the database connection open so that we can query the models in the tests, right?

In which case we probably want to make sure we're closing the connections in the `afterEach`

Reply:
yes, and good shout